site/docs/plugins/conversations.md

```diff
@@ -571,7 +571,64 @@ async function waitForMe(conversation, ctx) {
 
 An update can mean that a text message was sent, or a button was pressed, or something was edited, or virtually any other action was performed by the user.
 Check out the full list in the Telegram docs [here](https://core.telegram.org/bots/api#update).
+   
+The wait method always yields a new [context](https://grammy.dev/guide/context.html) object representing the received update.
+That means you're always dealing with as many context objects as there are updates received during the conversation.
 
+<CodeGroup>
+<CodeGroupItem title="TypeScript" active>
+
+```ts
+const TEAM_REVIEW_CHAT = -10018121912;
+async function askUser(conversation: MyConversation, ctx: MyContext) {
+  // Ask the user for their home address.
+  await ctx.reply("Could you state your home address?");
+  
+  // Wait for the user to send their address:
+  const userHomeAddressContext = await conversation.wait();
```

Comment:
Why context and not ctx?

Reply:
It's just easier to read, it's an example after all